Update coach's page with dates for fall 2017 workshop

daveliepmann opened the pull request.

daveliepmann committed:
Update 'Coach' page with fall 2017 dates

daveliepmann committed:
Add link to location of Wooga office

daveliepmann committed:
Wording improvements

daveliepmann committed:
Merge branch 'master' of https://github.com/clojurebridge-berlin/clojurebridge-berlin.github.io into update-dates-fall-2017

vsmart commented:
Looks good! Could you also update the Coach Form link on line `40` ? The new link is: https://goo.gl/forms/YMgTqhgQ0Ko1PNht1

daveliepmann committed:
Update Coach Form link

Per
https://github.com/clojurebridge-berlin/clojurebridge-berlin.github.io/pull/58#issuecomment-319758671
- ping @vsmart - assists with #58

daveliepmann commented:
Yes, done

vsmart commented:
Awesome! @daveliepmann \o/
